assets/app/views/builds.html

```diff
@@ -170,6 +187,9 @@
               Output image: {{build.parameters.output.dockerImageReference}}
             </div>            
           </div>
+          <div ng-show="true">
```

Comment:
actually no the check was on disabling the button, so i'm guessing this ng-show wasn't needed at all?